src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/util/HomeOfficeDateFormatter.java

```diff
@@ -17,4 +17,12 @@ public static String getCurrentDateTime() {
             DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'"));
     }
 
+    public static String getIacDecisionDate(String hoDecisionDate) {
```

Comment:
@divisathyan Could we not handle this conversion using date libraries than substring operations?

Reply:
We already get a proper dateTime string in the format "yyyy-MM-dd'T'HH:mm:ss'Z'"
I felt we could work with that assumption

Reply:
I am formatting the date. If there is any exception, then we return the data as sent by Home office. Confirmed with Damian and Ana